projects/texas/pelias.json

```diff
@@ -0,0 +1,224 @@
+      "import": [{
+        "filename": "texas-latest.osm.pbf"
+      }]
+    },
+    "openaddresses": {
+      "datapath": "/data/openaddresses",
+      "files": [
+        "us/tx/anderson.csv",
+        "us/tx/austin.csv",
+        "us/tx/bandera.csv",
+        "us/tx/bastrop.csv",
+        "us/tx/bexar.csv",
+        "us/tx/brazoria.csv",
+        "us/tx/brazos.csv",
+        "us/tx/burleson.csv",
+        "us/tx/camp.csv",
+        "us/tx/capcog.csv",
+        "us/tx/carson.csv",
+        "us/tx/city_of_abilene.csv",
+        "us/tx/city_of_alvin.csv",
+        "us/tx/city_of_amarillo.csv",
+        "us/tx/city_of_austin.csv",
+        "us/tx/city_of_beaumont.csv",
+        "us/tx/city_of_buda.csv",
+        "us/tx/city_of_carrollton.csv",
+        "us/tx/city_of_cedar_park.csv",
+        "us/tx/city_of_colleyville.csv",
+        "us/tx/city_of_dalhart.csv",
+        "us/tx/city_of_denison.csv",
+        "us/tx/city_of_denton.csv",
+        "us/tx/city_of_flower_mound.csv",
+        "us/tx/city_of_garland.csv",
+        "us/tx/city_of_horseshoe_bay.csv",
+        "us/tx/city_of_houston.csv",
+        "us/tx/city_of_hurst.csv",
+        "us/tx/city_of_killeen.csv",
+        "us/tx/city_of_lewisville.csv",
+        "us/tx/city_of_longview.csv",
+        "us/tx/city_of_mansfield.csv",
+        "us/tx/city_of_mckinney.csv",
+        "us/tx/city_of_palestine.csv",
+        "us/tx/city_of_pearland.csv",
+        "us/tx/city_of_pflugerville.csv",
+        "us/tx/city_of_rosenberg.csv",
+        "us/tx/city_of_san_antonio.csv",
+        "us/tx/city_of_sherman.csv",
+        "us/tx/city_of_southlake.csv",
+        "us/tx/city_of_sugar_land.csv",
+        "us/tx/city_of_temple.csv",
+        "us/tx/city_of_university_park.csv",
+        "us/tx/city_of_waco.csv",
+        "us/tx/city_of_waxahachie.csv",
+        "us/tx/colleyville.csv",
+        "us/tx/collin.csv",
+        "us/tx/colorado.csv",
+        "us/tx/comal.csv",
+        "us/tx/comanche.csv",
+        "us/tx/coryell.csv",
+        "us/tx/dallam.csv",
+        "us/tx/dallas.csv",
+        "us/tx/delta.csv",
+        "us/tx/denton.csv",
+        "us/tx/desoto.csv",
+        "us/tx/dimmit.csv",
+        "us/tx/duval.csv",
+        "us/tx/ector.csv",
+        "us/tx/el_paso.csv",
+        "us/tx/ellis.csv",
+        "us/tx/fannin.csv",
+        "us/tx/fayette.csv",
+        "us/tx/fisher.csv",
+        "us/tx/freestone.csv",
+        "us/tx/gaines.csv",
+        "us/tx/galveston.csv",
+        "us/tx/gillespie.csv",
+        "us/tx/gray.csv",
+        "us/tx/greater_houston.csv",
+        "us/tx/grimes.csv",
+        "us/tx/guadalupe.csv",
+        "us/tx/hansford.csv",
+        "us/tx/hartley.csv",
+        "us/tx/hays.csv",
+        "us/tx/hidalgo_county.csv",
+        "us/tx/hood.csv",
+        "us/tx/houston.csv",
+        "us/tx/howard.csv",
+        "us/tx/hunt.csv",
+        "us/tx/johnson.csv",
+        "us/tx/kaufman.csv",
+        "us/tx/keller.csv",
```

Comment:
"us/tx/keller.csv", does not appear to exist in the openaddresses list
![image](https://user-images.githubusercontent.com/2505982/83835502-8e4ea980-a6b6-11ea-8a29-61ad22e33b8b.png)